great_expectations/datasource/fluent/spark_file_path_datasource.py

```diff
@@ -20,12 +21,17 @@
 class CSVAsset(_FilePathDataAsset):
     # Overridden inherited instance fields
     type: Literal["csv"] = "csv"
+    header: bool = False
+    infer_schema: bool = False
+
+    class Config:
+        extra = pydantic.Extra.forbid
 
     def _get_reader_method(self) -> str:
         return self.type
 
     def _get_reader_options_include(self) -> set[str] | None:
-        return {"header", "inferSchema"}
+        return {"header", "infer_schema"}
```

Comment:
We were inconsistently using `inferSchema` and `inferer_schema`.
`infer_schema` seems to be correct based on my local testing of the latest pyspark but I haven't tested other versions.

Reply:
@Kilo59 Please note that `header` and `inferSchema` are options for `PySpark` that are not supposed to be hardcoded (often they need to be `False`).  Thanks.